package.json

```diff
@@ -55,11 +56,11 @@
     "shelljs": "0.3.0",
     "tabtab": "https://github.com/Icenium/node-tabtab/tarball/master",
     "temp": "0.8.1",
-    "unzip": "0.1.9",
+    "unzip": "0.1.11",
```

Comment:
Is this change required - do we use unzip anywhere? If not, let's just remove it.